packages/block-library/src/buttons/style.scss

```diff
@@ -13,3 +13,13 @@
 .wp-block-buttons.aligncenter {
 	text-align: center;
 }
+
+.wp-block-buttons.is-direction-vertical .wp-block-button {
+	display: block;
+	width: max-content;
```

Comment:
`max-content` isn't supported in IE11, so just wondering how this looks in IE?

Reply:
I noticed that if you remove `width: max-content`, `margin-left: auto;` and `margin-right: auto;` the blocks seem to center almost correctly because `block-editor-inner-blocks` is a flex-item. 

I think similar to what I mentioned above for the editor styles, `margin-left` and `margin-right` should probably be reset to `0px` for all instances of a vertical button. I don't think any margin would be needed whether the buttons are unaligned, centered, left, or right.

Reply:
I was actually concerned if `max-content` would work as well. I actually used that after seeing it being used [here](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-list/style.scss#L569). I've implemented the suggested changes. Thank you! ❤️